components/mdns/mdns_mem_caps.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <string.h>
```

Comment:
Nit: please inlcude the header file `sdkconfig.h` since the macro `CONFIG_XXX` is used in this file.

Reply:
good point -- fixed, thanks!